src/_listBase.scss

```diff
@@ -123,6 +137,14 @@
     &:last-child:first-child {
       flex-basis: auto;
     }
+
+    &--growing {
+      @include fd-flex-center();
+    }
+
+    &--unread {
+      font-weight: bold;
+    }
```

Comment:
where are these in the html code? I looked for `__title--growing` and `__title--unread` but couldn't find it. If it's present in NGX but not in styles, you should add examples in fund-styles so we can test it.

Reply:
added examples.

Reply:
removed these duplicate classes